src/App.jsx

```diff
@@ -26,15 +28,41 @@ const OwnerPage = lazy(() => import('./pages/OwnerPage'))
 const PullRequestPage = lazy(() => import('./pages/PullRequestPage'))
 const RepoPage = lazy(() => import('./pages/RepoPage'))
 
+const SetUpActions = Object.freeze({
```

Comment:
Optional: I find this and the HomePageRedirect component could be in a different file. Feels it's taking away from what the app page is meant to represent